pkg/kubelet/kubelet.go

```diff
@@ -1409,6 +1426,12 @@ func (kl *Kubelet) Run(updates <-chan kubetypes.PodUpdate) {
 		if utilfeature.DefaultFeatureGate.Enabled(features.NodeLease) {
 			go kl.nodeLeaseController.Run(wait.NeverStop)
 		}
+
+		// start informer for CSIDriver
+		if utilfeature.DefaultFeatureGate.Enabled(features.CSIDriverRegistry) {
+			go kl.informerFactory.Start(wait.NeverStop)
+			go kl.informerFactory.WaitForCacheSync(wait.NeverStop)
```

Comment:
I think we need to wait for cache sync synchronously.  But maybe we should do this inside volumeManager.Run instead of here since it doesn't seem like anything else in kubelet is using the shared informer.  We only need to have volumeManager wait for the informer sync.

Reply:
Ok